spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasModule.scala

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
```

Comment:
Instead of a subclass of AbstractModule, I think we need a KerasStyleModuleBuilder. The ModuleBuilder will build a module when it is added into a container or connected into a graph

Reply:
`Keras` layer should be a subclass of `AbstractModule` in order to mix with the existing layer.

Reply:
I don't think we need a Keras layer. I think a KeraBuilder which produce AbstarctModule is enough

Reply:
You just need to add a new method

Container.add(builder: KerasBuilder:) {
   infer shape
   use keras builder to return a module
}

Module.inputs(builder: KerasBuilder) : {
    get output shape
    user kerasBuilder to build module
}